runtime/runtime-core/common/test/aws/smithy/kotlin/runtime/io/GzipByteReadChannelTest.kt

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.io
+
+import aws.smithy.kotlin.runtime.hashing.crc32
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+import kotlin.test.assertContentEquals
+import kotlin.test.assertEquals
+
+expect fun decompressGzipBytes(bytes: ByteArray): ByteArray
```

Comment:
**nit**: Move to it's own file in both common and JVM test since it is re-used in more than one test.

e.g.

```
// file: GzipTestUtils.kt
expect fun decompressGzipBytes(bytes: ByteArray): ByteArray

// file: GzipTestUtilsJVM.kt
actual fun decompressGzipBytes(bytes: ByteArray): ByteArray
```

Reply:
Should this be `InternalApi`?

Reply:
No it's in a test sourceset and isn't published, it's only applicable to tests.